packages/babel-plugin-proposal-private-methods/test/fixtures/accessors/tagged-template/exec.js

```diff
@@ -0,0 +1,10 @@
+class Foo {
+  get #tag() {
+    return () => this;
```

Comment:
I screwed this up. It should be `return function() { return this };`, since an arrow function will only return the `this` context from the `get #tag`, not the invocation of the returned function.
```suggestion
    return function() { return this; };
```